server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -101,7 +101,7 @@ class LivyServer extends Logging {
       )
       val launch_keytab = livyConf.get(LAUNCH_KERBEROS_KEYTAB)
       val launch_principal = SecurityUtil.getServerPrincipal(
-        livyConf.get(LAUNCH_KERBEROS_PRINCIPAL), server.host)
+        livyConf.get(LAUNCH_KERBEROS_PRINCIPAL), host)
```

Comment:
AFAIK kerberos requires FQDN, and here host is "0.0.0.0" by default, should we use a hostname instead?